tests/test___init__.py

```diff
@@ -249,3 +250,19 @@ def test_copy():
         before = helper._crc
         helper.update(b"FACEDACE")
         assert clone._crc == before
+
+    @staticmethod
+    @pytest.mark.parametrize("chunksize", [1, 3, 5, 7, 11, 13, ISCSI_LENGTH])
+    def test_consume_stream(chunksize):
+        helper = crc32c.Checksum()
+        expected = list(iscsi_chunks(chunksize))
+        stream = mock.Mock(spec=["read"])
+        stream.read.side_effect = expected + [""]
+
+        found = list(helper.consume(stream, chunksize))
+
+        assert helper._crc == ISCSI_CRC
+        assert found == expected
+        for call in stream.read.call_args_list:
+            assert call[0] == (chunksize,)
+            assert call[1] == {}
```

Comment:
Instead:

```python
assert call == mock.call(chunksize)
```

I'm always uneasy about indexing into `mock.call()` objects because it isn't "exhaustive" the way `==` is.